### PR TITLE
Update NDK (Android NDK r25 LTS)

### PR DESCRIPTION
**Pull request by AGulev**

### CODE REVIEW

File: server/docker-base/Dockerfile [hunk 1]
```diff
@@ -221,16 +221,20 @@ ENV ANDROID_MIN_API_LEVEL 16
 ENV ANDROID_GCC_VERSION 4.8
 ENV ANDROID_SDK_VERSION 31
 ENV ANDROID_SDK_FILENAME android-sdk-linux-android-${ANDROID_SDK_VERSION}-${ANDROID_BUILD_TOOLS_VERSION}.tar.gz
-ENV ANDROID_STL_INCLUDE ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/sources/cxx-stl/gnu-libstdc++/${ANDROID_GCC_VERSION}/include
-ENV ANDROID_STL_ARCH_INCLUDE ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/sources/cxx-stl/gnu-libstdc++/${ANDROID_GCC_VERSION}/libs/armeabi-v7a/include
-ENV ANDROID_STL_LIB ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/sources/cxx-stl/gnu-libstdc++/${ANDROID_GCC_VERSION}/libs/armeabi-v7a
-ENV ANDROID_SYSROOT ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/platforms/android-${ANDROID_NDK_API_VERSION}/arch-arm
-ENV ANDROID_BIN_PATH ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/toolchains/arm-linux-androideabi-${ANDROID_GCC_VERSION}/prebuilt/linux-x86_64/bin
```
Comment (AGulev) [inline server/docker-base/Dockerfile:228]: Old unused variables (I checked 2 y o dev - there is no such bars in it)

File: server/docker-base/Dockerfile [hunk 2]
```diff
@@ -221,16 +221,20 @@ ENV ANDROID_MIN_API_LEVEL 16
...
+
 ENV ANDROID_SDK_BUILD_TOOLS_PATH ${ANDROID_HOME}/build-tools/${ANDROID_BUILD_TOOLS_VERSION}
 ENV ANDROID_LIBRARYJAR ${ANDROID_HOME}/platforms/android-${ANDROID_SDK_VERSION}/android.jar
 
-# We must keep two NDKs alive for now, since migrating to the latest (i.e 20) will break
+# We must keep two NDKs alive for now, since migrating to the latest (i.e 25b) will break
 # for users that try to build with an older build.yml.
+
+ENV AAPT ${ANDROID_SDK_BUILD_TOOLS_PATH}/aapt
+ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/platform-tools:${ANDROID_SDK_BUILD_TOOLS_PATH}
```
Comment (AGulev) [inline server/docker-base/Dockerfile:232]: SDK separated from NDK for easier maintainance

File: server/docker-base/Dockerfile [hunk 3]
```diff
@@ -216,21 +216,22 @@ ENV ANDROID_HOME ${ANDROID_ROOT}/android-sdk-linux
...
-ENV ANDROID_BIN_PATH ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK_VERSION}/toolchains/arm-linux-androideabi-${ANDROID_GCC_VERSION}/prebuilt/linux-x86_64/bin
+
 ENV ANDROID_SDK_BUILD_TOOLS_PATH ${ANDROID_HOME}/build-tools/${ANDROID_BUILD_TOOLS_VERSION}
 ENV ANDROID_LIBRARYJAR ${ANDROID_HOME}/platforms/android-${ANDROID_SDK_VERSION}/android.jar
 
-# We must keep two NDKs alive for now, since migrating to the latest (i.e 20) will break
+# We must keep two NDKs alive for now, since migrating to the latest (i.e 25b) will break
 # for users that try to build with an older build.yml.
+
+ENV AAPT ${ANDROID_SDK_BUILD_TOOLS_PATH}/aapt
```
Comment (JCash) [inline server/docker-base/Dockerfile:228]: Probably good to add a version number here? E.g. AAPT_sdkversion

File: server/docker-base/Dockerfile [hunk 4]
```diff
@@ -243,9 +244,27 @@ ENV ANDROID_NDK20_STL_LIB    ${ANDROID_NDK20_PATH}/sources/cxx-stl/llvm-libc++/l
...
+ENV ANDROID_64_NDK25_API_VERSION 21
+
+# These paths are the same for both the 32 and 64 bit toolchains
+ENV ANDROID_NDK25_PATH       ${ANDROID_ROOT}/android-ndk-r${ANDROID_NDK25_VERSION}
+ENV ANDROID_NDK25_BIN_PATH   ${ANDROID_NDK25_PATH}/toolchains/llvm/prebuilt/linux-x86_64/bin
+ENV ANDROID_NDK25_SYSROOT    ${ANDROID_NDK25_PATH}/toolchains/llvm/prebuilt/linux-x86_64/sysroot
+ENV ANDROID_NDK25_FILENAME   android-ndk-r${ANDROID_NDK25_VERSION}-linux.tar.gz
+
+# We specify it in build_input.yml by setting it the first in PATH
+# ENV PATH ${PATH}:${ANDROID_NDK25_BIN_PATH}
```
Comment (JCash) [inline server/docker-base/Dockerfile:264]: 👍

**Review by JCash — APPROVED** [review]

**Pull request merged**